src/plugins/enum/README.md

```diff
@@ -0,0 +1,19 @@
+- infos = Information about the enum plugin is in keys below
+- infos/author = Thomas Waser <>
+- infos/licence = BSD
+- infos/needs =
+- infos/provides = check
+- infos/placements = presetstorage
+- infos/description =
+
+## Introduction ##
+
+The Enum plugin checks string values of Keys by comparing it against a list of valid values.
+
+## Usage ##
+
+The plugin checks every Key in the Keyset for the Metakey `check/enum` containing a list with the syntax `['string1', 'string2', 'string3', ..., 'stringN']` and compares each value with the string value of the Key. If no match is found an error is returned.
```

Comment:
> By the <> part i meant something like:
> system/configs/enums/valid1='a','b','c'
> system/configs/enums/valid2='g','h','i'
> 
> And the check/enum meta key could look like <enums/valid1>,
> 'd','e','f',<enums/valid2>
> 
> So the plugin would check for a,b,c,d,e,f,g,h

I would prefer to have as little syntactic clutter as possible (aka not having 
`<>`). Syntax should be introduced in the storage plugins, not within values.


One could even argue to introduce an array:

```
check/enum
check/enum/#0=a
check/enum/#1=b
check/enum/#1=c
```

Where a, b and c are literals, e.g. serialized in json:

```
{ "check" : { "enum" : ["a", "b", "c"] } }
```

Obviously, the data structure would be huge, we would not have nice references 
(without introducing syntax) and you would need to reimplement most things.
So the `'a', 'b'` for now and `var, 'a', 'b', "'"` for later seems like a good 
middle ground.